composer/loggers/slack_logger.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Callable, Dict, Optional
+
+from slack_sdk import WebhookClient
+
+from composer.loggers.logger_destination import LoggerDestination
+
+
+class SlackLogger(LoggerDestination):
```

Comment:
Should we add some `filter_keys` option or something? Otherwise this is going to log _everything_ to slack right?

Reply:
done!